src/components/Profile/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {useSelector} from "react-redux";
+import {userInfoSelector} from "../../state/user/user.selectors";
+
+export default function Profile() {
+    let user = useSelector(state => userInfoSelector(state));
+    return (
+        <div class="mt-3 mb-3 h-100">
```

Comment:
@nikhilrayaprolu
mt-3 & mb-3 should be changed to a class that set both of TOP and BOTTOM
Please take a look at here - https://mdbootstrap.com/docs/react/utilities/spacing/